Scheduler recurrenceException: edit description and the Appointment Types article (#1510)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/dxSchedulerAppointment/recurrenceException.md

---
---
##### shortDescription

> Specifies the start date and time of one or more appointments to delete from a series. Requires [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) to be set.

Reviewer: ```suggestion
Specifies the start date and time of one or more appointments to exclude from a series. This option requires that you also set [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule).
```

---

> In the example below, the appointment takes place daily except Fabruary 22 and 23, 2021. Note that **recurrenceException** sets time in UTC as well as [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate) and [endDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate):

Reviewer: ```suggestion
In the example below, the appointment takes place daily except February 22 and 23, 2021. The **recurrenceException** option requires that you specify time, [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate), and [endDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate) in the UTC format:
```

    {
        text: 'Daily planning',
        startDate: new Date('2021-02-20T07:00:00.000Z'),
        endDate: new Date('2021-02-20T08:00:00.000Z'),
        recurrenceRule: 'FREQ=DIALY',
        recurrenceException: '20210222T070000Z,20210223T070000Z'

>     }

Reviewer: I suggest you to write a full snippet for this for all platforms. Fix also 'FREQ=DIALY', please.

---
---
##### shortDescription

> Specifies the start date and time of one or more appointments to exclude from a series. This option requires that you also set [recurrenceRule]/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule).

Reviewer 2: The link doesn't render.
**Always run the website locally and check that everything looks as it should. No exceptions.**

---

> For an example on how to sprecify an exception, refer to the [Appointment Types](/concepts/05%20Widgets/Scheduler/030%20Appointments/015%20Appointment%20Types/030%20Recurring%20Appointments.md '/Documentation/Guide/Widgets/Scheduler/Appointments/Appointment_Types/#Recurring_Appointments') article.

Reviewer 2: ```suggestion
For an example of how to specify an exception, refer to the [Appointment Types](/concepts/05%20Widgets/Scheduler/030%20Appointments/015%20Appointment%20Types/030%20Recurring%20Appointments.md '/Documentation/Guide/Widgets/Scheduler/Appointments/Appointment_Types/#Recurring_Appointments') article.
```

> For an example on how to sprecify an exception, refer to the [Appointment Types](/concepts/05%20Widgets/Scheduler/030%20Appointments/015%20Appointment%20Types/030%20Recurring%20Appointments.md '/Documentation/Guide/Widgets/Scheduler/Appointments/Appointment_Types/#Recurring_Appointments') article.

Reviewer 2: Shouldn't it be "refer to the Recurring Appointments article"?
